.pipelines/singletenancy/azure-cni-overlay/azure-cni-overlay-e2e-step-template.yaml

```diff
@@ -68,14 +68,7 @@ steps:
 
   - script: |
       echo "Start Integration Tests on Overlay Cluster"
-      echo "deploy ip-masq-agent for overlay"
-      kubectl apply -f test/integration/manifests/ip-masq-agent/ip-masq-agent.yaml --validate=false
-      cd test/integration/manifests/ip-masq-agent/
-      kubectl apply -f config-custom.yaml
-      kubectl apply -f config-reconcile.yaml
-      cd ../../../..
-      kubectl get po -owide -A
-      CNS=$(make cns-version) DROPGZ=$(make cni-dropgz-version)
+      CNS=$(make cns-version)
```

Comment:
nit: probably we can add CNS=$(make cns-version) to line 72? CNS_VERSION=$(make cns-version)

Reply:
Yeah probably, wondering if nested `make` calls would be an issue

Reply:
maybe that's a question for another PR

Reply:
Nested make commands run in serial and creates another shell until the command is completed. If it fails, the exit code is passed along the stack. The only problem with make commands is if you do not use `set -e` OR if another command replaces the exit code before the `- script: |` block is done then pipelines will not catch the failure.